pennylane/transforms/__init__.py

```diff
@@ -44,15 +44,26 @@
     :toctree: api
 
     ~adjoint
+    ~ctrl
+    ~transforms.invisible
+    ~apply_controlled_Q
+    ~quantum_monte_carlo
+
+Transforms for circuit compilation and optimization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This set of transforms accept quantum functions, and perform basic circuit
+optimization tasks.
+
+.. autosummary::
+    :toctree: api
+
+    ~compile
```

Comment:
🙌 

Now that the organization of the docs is clearer, perhaps this should inform the code optimization? From reading the docs, it sounds like this section belongs in a single folder

Reply:
e.g., `transforms/optimization/{compile,cancel_inverses,commute_controlled, etc.}`

Reply:
Currently everything except `compile` is in a separate folder (`optimization`). I kept `compile` out because if we eventually want to have, e.g., `synthesis` or `transpilation` folders, we'd be pulling stuff from all of them to use in `compile`.